universe/Universe.cpp

```diff
@@ -683,47 +683,60 @@ void Universe::InitMeterEstimatesAndDiscrepancies() {
     }
 }
 
-void Universe::UpdateMeterEstimates()
-{ UpdateMeterEstimates(INVALID_OBJECT_ID, false); }
-
-void Universe::UpdateMeterEstimates(int object_id, bool update_contained_objects) {
-    if (object_id == INVALID_OBJECT_ID) {
-        for (int obj_id : m_objects.FindExistingObjectIDs())
-            m_effect_accounting_map[obj_id].clear();
-        // update meters for all objects.  Value of updated_contained_objects is irrelivant and is ignored in this case.
-        UpdateMeterEstimatesImpl(std::vector<int>());// will cause it to process all existing objects
-        return;
-    }
-
-    // collect objects to update meter for.  this may be a single object, a group of related objects, or all objects
-    // in the (known) universe.  also clear effect accounting for meters that are to be updated.
-    std::set<int> objects_set;
-    std::list<int> objects_list;
-    objects_list.push_back(object_id);
+void Universe::UpdateMeterEstimates() {
+    for (int obj_id : m_objects.FindExistingObjectIDs())
+        m_effect_accounting_map[obj_id].clear();
+    // update meters for all objects.
+    UpdateMeterEstimatesImpl(std::vector<int>());
+}
+
+void Universe::UpdateMeterEstimates(int object_id, bool is_update_contained_objects) {
```

Comment:
Why the "is_" in the parameter name?

Reply:
Removed.

The "is_" is there so that the expression `if (is_X)` reads correctly.  However, `if (update_contained_objects)` is better.